backend/src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -65,6 +65,38 @@ public boolean containsIngredient(String ingredient) {
     return false;
   }
 
+  public String getName() {
```

Comment:
nit: looks like these getters are only used for tests and so we should annotate with @VisibleForTesting